src/dart_dll.cpp

```diff
@@ -175,7 +180,7 @@ Dart_Handle DartDll_RunMain(Dart_Handle library) {
       Dart_Invoke(isolateLib, Dart_NewStringFromCString("_startMainIsolate"),
                   kNumIsolateArgs, isolateArgs);
   if (Dart_IsError(result)) {
-    std::cout << "Dart initialized, error was: " << Dart_GetError(result)
+    std::cout << "Dart initialized, error was: " << Dart_GetError(result) << "\n"
```

Comment:
Why change this?

Reply:
As it was causing output to concat on the same line. I might make a follow up pr that has a logger interface instead as well as want to avoid uncontrolled std::cout in my use case.